features/promotion/managing_promotions/adding_promotion_with_action.feature

```diff
@@ -10,8 +10,8 @@ Feature: Adding a new promotion with action
 
     @ui @javascript
     Scenario: Adding a new promotion with fixed discount
-        Given I want to create a new promotion
-        When I specify its code as "10_for_all_products"
+        When I want to create a new promotion
+        And I specify its code as "10_for_all_products"
```

Comment:
I would say that `Given` looks fine for me for all steps starting with "I want to". We open a page in this step, so it is like a preparation of an initial state 😉